src/status_im/ui/screens/profile/seed/views.cljs

```diff
@@ -36,7 +36,9 @@
 
 (defn intro []
   [react/view {:style styles/intro-container}
-   [components.common/image-contain {:style styles/intro-image}
+   [components.common/image-contain {:container-style styles/intro-image
+                                     :width           993
```

Comment:
Magic number?

Reply:
It's width and height of the source image